src/app/rundown/components/segment-end-indicator/segment-end-indicator.component.html

```diff
@@ -0,0 +1,11 @@
+<sofie-timeline-flag
+    [label]="lastPartInSegment?.autoNext ? 'AUTO' : (lastPartInSegment?.isOnAir ? 'NEXT' : undefined)"
```

Comment:
Don't know if it can be done any better in Angulars Html templates, but nesting ternaries are something we don't like, as far as i know.